src/index.ts

```diff
@@ -16,4 +19,8 @@ const server = new GraphQLServer({
   typeDefs,
   resolvers,
 })
-server.start(() => console.log(`Server is running on http://localhost:4000`))
+server.start(() => console.info(`Server is running on http://localhost:4000`))
+
+createConnection().then(async connection => {
```

Comment:
Faz sentido manter o servidor de pé se a conexão com o banco falhou? Não seria melhor condicionar o início do servidor mediante a conexão com o banco?

Reply:
Concordo, faria bem mais sentido iniciar o servidor quando a conexão com o banco for feita.